src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/health/FeatureSwitchInfoContributor.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.health
+
+import org.springframework.boot.actuate.info.Info
+import org.springframework.boot.actuate.info.InfoContributor
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.core.ServiceAgencySwitchesService
+
+@Component
+class FeatureSwitchInfoContributor(
+  private val serviceAgencySwitchesService: ServiceAgencySwitchesService,
+) : InfoContributor {
+
+  @Transactional(readOnly = true)
```

Comment:
this looks a bit odd - especially as it is calling a different service anyway.  Would have expected the `readonly = true` to be on that service instead?  Perhaps default at the top of the file should be `readonly` and then changed as needed for methods that amend data?